examples/fluids/navierstokes.c

```diff
@@ -1149,7 +1149,7 @@ int main(int argc, char **argv) {
     CHKERRQ(ierr);
   }
   ierr = PetscStrncpy(user->outputfolder, ".", 2); CHKERRQ(ierr);
```

Comment:
What is this line for? I can't understand the purpose here!

Reply:
It's setting the default output to the current directory. A file name formatted with `"%s/%s.vtu"` thus becomes `./*.vtu` instead of `some-dir/*.vtu`.

An alternative would be to initialize it blank and ensure that if a directory is specified, it ends with `/`. Then format with `"%s%s.vtu"` and you'll get functionally the same behavior with `*.vtu` and `some-dir/*.vtu`.

Reply:
Ah, it makes sense! Thanks. What we have right now looks good. I added the same line to the solids example as well.